Xamarin.PropertyEditing/ViewModels/ObjectPropertyViewModel.cs

```diff
@@ -107,11 +107,10 @@ private set
 			}
 		}
 
-		public ValueSource ValueSource
+		public new ValueSource ValueSource
```

Comment:
Instead of using `new` here and making this now an ambiguous second version of a property on the base class a better solution would be to move the `private set` outside the property into a separate method. Since it's private here we don't actually need it as part of the property; the mental cost of having it a separate (ex.) `SetValueSource` method is lower than that of tracking two separate versions of the same property.

`new` should almost never be used on member declarations, its usage should be considered very carefully.

Reply:
The original code from master didn't have the `new` there, so was added to remove the compiler warning without refactoring the `private set` to another method. I'll refactor it out now we're making the base property abstract.